tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -8,19 +8,21 @@
     <suppress files=".+[\\/]internal[\\/].+\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
     <suppress files=".+DTO\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck" />
     <suppress files=".+Impl\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
-	<suppress files=".+\\pom\.xml" checks="OnlyTabIdentationInXmlFilesCheck"/>
+    <suppress files=".+\\pom\.xml" checks="OnlyTabIndentationInXmlFilesCheck"/>
```

Comment:
Since you are touching this line you might as well also apply this fix from https://github.com/openhab/static-code-analysis/pull/289 here to remove these false positives for people with Unix paths.

Reply:
@wborn Done.